### DISCUSSION TITLE
Settings missing - Doubleclick close tab and Rename select filename onle

### DISCUSSION THREAD

**Discussion by francwalter** [doublecmd/doublecmd#865] [General]
Hye

I do not find two settings which I am used from the older brother, Total Commander:
1. double click on a tab closes the tab, I think there is a setting for it in totalcmd but in DC I dont find it.
2. the rename of file (F2) does select the whole file **and** the extension name, not only the filename. Also here I think there is a setting in totalcmd to select with F2 only the file name (useful if you dont want to change the extension but the file name, which is mostly the case).

Where are these settings?
Possible to add them settings if the dont exist?
Thanky,
franky

DC 1.0.5 beta Rev. 104 on Ubuntu 22.04

**Comment by alexx2000** [comment]
Hi
1. [Action to do when double click on a tab](https://doublecmd.github.io/doc/en/configuration.html#ConfigTabs)
2. [Select file name without extension when renaming](https://doublecmd.github.io/doc/en/configuration.html#ConfigOperations)

**Comment by francwalter** [comment]
Exactly that :smile: 
Thank a lot!
